tests/test_parser.py

```diff
@@ -323,6 +320,13 @@ def test_login_page_has_expected_title(self):
         actual = validate_title(html)
         self.assertTrue(actual)
 
+    def test_childs_page_has_expected_username(self):
+        response_string = '\n<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" \
+                     "http://www.w3.org/TR/html4/strict.dtd">\n<HTML>\n \
+                     <HEAD>\n<TITLE>login</TITLE>\n'
+        self.assertTrue(childs_page_return_right_login(response_string,
+                                                       'login'))
+
 
```

Comment:
there is no negative test. what will happen if we pass invalid value?

Reply:
If this way is OK, i add negative test